x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_instances/agent_contextual_information/index.tsx

```diff
@@ -112,8 +129,25 @@ export function AgentContextualInformation({
           </EuiFlexItem>
         </EuiFlexGroup>
       ),
-      width: '25%',
+      width,
     },
+    ...(latestAgentVersionEnabled
+      ? [
+          {
+            label: latestVersionLabel,
+            fieldName: latestVersionLabel,
+            val: (
+              <AgentLatestVersion
+                agentName={agentName as AgentName}
```

Comment:
don't think this is necessary

Reply:
Addressed in [7b2cefb](https://github.com/elastic/kibana/pull/153643/commits/7b2cefb4943bce8e0da6cb80b547d1f3a0d48db4).